### PR TITLE
Update JS examples to follow new JS quickstart installation instructions

### PR DESCRIPTION
**Pull request by alexisintech**
This PR updates JS examples to include appropriate installation/Clerk instantiation instructions. It injects the Frontend API URL into `<script>` ClerkJS instantiation. It also removes installations instructions from the JS reference overview, and points users to the quickstart for detailed instructions.

### CODE REVIEW

File: docs/advanced-usage/using-proxies.mdx [hunk 1]
```diff
@@ -140,94 +140,126 @@ In order to enable proxying, you need to set a proxy URL for your Clerk instance
...
-                <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-              </Head>
-              <Component {...pageProps} />
-            </ClerkProvider>
-          );
-        }
-      ```
+    To configure your proxy setup using properties in your Next.js application, set the `proxyUrl` property on the [`<ClerkProvider>`](https://clerk.com/docs/components/clerk-provider) component.
+
+    ```tsx filename="_app.tsx"
```
Comment (S3Prototype) [inline docs/advanced-usage/using-proxies.mdx:181]: Should we also add an app router example here?

File: docs/advanced-usage/using-proxies.mdx [hunk 2]
```diff
@@ -140,94 +140,126 @@ In order to enable proxying, you need to set a proxy URL for your Clerk instance
...
+      });
+    ```
+  </Tab>
 
-        export default ClerkApp(App, {
-          proxyUrl: 'https://app.dev/__clerk'
-        });
-      ```
+  <Tab>
+    To configure your proxy setup using properties in your JavaScript application, pass the `proxyUrl` option to the [`load()`](https://clerk.com/docs/references/javascript/clerk/clerk#load) method.
```
Comment (S3Prototype) [inline docs/advanced-usage/using-proxies.mdx:227]:
```suggestion
    To configure your proxy setup using properties in your JavaScript application, pass the `proxyUrl` option to the [`load()`](/docs/references/javascript/clerk/clerk#load) method.
```

File: docs/components/authentication/sign-in.mdx [hunk 3]
```diff
@@ -153,6 +153,16 @@ function mountSignIn(node: HTMLDivElement, props?: SignInProps): void;
   <!-- Add a <div id="sign-in"> element to your HTML -->
   <div id="sign-in"></div>
 
+  <!-- Initialize Clerk with your
+  Clerk Publishable key and Frontend API URL -->
+  <script
```
Comment (S3Prototype) [inline docs/components/authentication/sign-in.mdx:158]: Highlighted line in this mountSignIn example is off now

Comment (S3Prototype) [reply docs/components/authentication/sign-in.mdx:158]: Actually, all the highlighted lines on this page seem to be off

File: docs/custom-flows/user-impersonation.mdx [hunk 4]
```diff
@@ -102,79 +102,121 @@ The above JSON snippet shows what a Clerk session token looks like when a user i
 
 When working with frontend (client-side) code, you can easily detect impersonated sessions and gain access to the actor ID or other information from the `act` claim.
```
Comment (S3Prototype) [inline docs/custom-flows/user-impersonation.mdx:103]:
```suggestion
When working with client-side code, you can detect impersonated sessions and gain access to the actor ID or other information from the `act` claim.
```
I know you didn't change this line, but it's worth cleaning up a little for a quick win

File: docs/custom-flows/user-impersonation.mdx [hunk 5]
```diff
@@ -102,79 +102,121 @@ The above JSON snippet shows what a Clerk session token looks like when a user i
...
-import { useAuth } from "@clerk/nextjs";
-import { withServerSideAuth } from "@clerk/nextjs";
-
-// SSR authentication context
-export const getServerSideProps = withServerSideAuth(
-  async ({ req }) => {
-    // The request is augmented with an auth object.
-    // The actor object is available in there.
-    const { userId, actor } = req.auth;
+<Tabs type="framework" items={["Next.js", "React", "JavaScript"]}>
```
Comment (S3Prototype) [inline docs/custom-flows/user-impersonation.mdx:105]: Is it worth adding an app router example here?

File: docs/components/authentication/sign-in.mdx (hunk 3 above)
Comment (alexisintech) [reply docs/components/authentication/sign-in.mdx:158]: oop i'll fix all those

File: docs/references/javascript/overview.mdx [hunk 6]
```diff
@@ -11,88 +11,13 @@ While we typically recommend using one of our framework SDK's, such as [Clerk Re
...
-    await window.Clerk.load({
-      // Set load options here...
-    });
-  });
-  document.body.appendChild(script);
-</script>
-```
-
-</InjectKeys>
+Please follow the instructions in the [ClerkJS quickstart](/docs/quickstarts/javascript) to add ClerkJS to your project.
```
Comment (S3Prototype) [inline docs/references/javascript/overview.mdx:14]: Love it

File: docs/custom-flows/user-impersonation.mdx (hunk 5 above)
Comment (alexisintech) [reply docs/custom-flows/user-impersonation.mdx:105]: yesss, we have slowly but surely been adding app router examples to the docs ([DOCS-259](https://linear.app/clerk/issue/DOCS-259/app-router-code-missing-for-a-lot-of-nextjs-examples))

File: docs/components/authentication/sign-in.mdx (hunk 3 above)
Comment (S3Prototype) [reply docs/components/authentication/sign-in.mdx:158]: Maybe we should just drop the line highlights. I don't personally find them necessary in these specific examples, and they cost of maintaining them, in aggregate, is probably higher than the value of having them in this case, IMO.

**Review by S3Prototype — CHANGES_REQUESTED** [review]
Looks good. Just let me know what you think of:
1. Adding app router examples. Maybe they're unnecessary, or we can tackle them in future and just get this PR through now.
1. Removing or updating the line highlights

**Pull request merged**